TYPO3.Neos/Classes/TYPO3/Neos/Domain/Service/UserService.php

```diff
@@ -749,7 +750,7 @@ protected function getAllRoles(User $user)
      */
     protected function createPersonalWorkspace(User $user, Account $account)
     {
-        $userWorkspaceName = 'user-' . $account->getAccountIdentifier();
+        $userWorkspaceName = 'user-' . UserUtility::slugifyUsername($account->getAccountIdentifier());
```

Comment:
Maybe a new method in the WorkspaceRepository like findByAcountIdentifier can be a good idea to avoid future bug, like forgetting to use the slugify method